index.html

```diff
@@ -13,50 +13,152 @@
 <body>
   <header>
     <div><a href="#" ><object data="./svgfiles/image_1.svg" type="image/svg+xml" ></object></a></div>
-    <nav><object class="menuimg" data="./svgfiles/ic_menu.svg" type="image/svg+xml" ></object></nav>
+    <nav class="mobile--nav"><object class="menuimg" data="./svgfiles/ic_menu.svg" type="image/svg+xml" ></object></nav>
+    <nav class="desktop--nav">
+      <ul class="desktop--nav--ul">
+        <li>Hello  <hr > </li>
```

Comment:
- [ ] please the desktop links should show just in the desktop version
![Screenshot (838)_LI](https://user-images.githubusercontent.com/65298048/221126691-955f9022-30c6-41ec-8b42-98123429731b.jpg)